src/sentry/migrations/0177_drop_targets.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 1.11.29 on 2021-03-12 17:46
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0176_remove_targets"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                        ALTER TABLE "sentry_notificationsetting" DROP COLUMN "target_identifier";
+                        ALTER TABLE "sentry_notificationsetting" DROP COLUMN "target_type";
+                        """,
+                    reverse_sql="""
```

Comment:
Since there are no rows should I remove this or does it not matter? If I do, is empty string needed or just remove the `reverse_sql` statement entirely?

I also wasn't sure whether int was correct since it was `BoundedPositiveIntegerField` and `BoundedBigIntegerField` types. Maybe I want bigint for the former.

Reply:
We only really use `reverse_sql` in dev, it's just convenience in case someone needs to roll back to test something, so it's generally good to include. I'd add `target_identifier` as `bigint`, other than that you're good.